src/communication/azext_communication/manual/_params.py

```diff
@@ -272,7 +272,3 @@ def _load_email_arguments(self):
                    ' Required for each attachment. Known values are: avi, bmp, doc, docm,'
                    ' docx, gif, jpeg, mp3, one, pdf, png, ppsm, ppsx, ppt, pptm, pptx,'
                    ' pub, rpmsg, rtf, tif, txt, vsd, wav, wma, xls, xlsb, xlsm, and xlsx')
-
-    with self.argument_context('communication email status get') as c:
-        c.argument('message_id', options_list=['--message-id'], type=str,
-                   help='System generated message id (GUID) returned from a previous call to send email')
```

Comment:
is the get operation status not needed?

Reply:
With LRO, we no longer have a separate call to get message status.